styles/main.scss

```diff
@@ -395,3 +397,94 @@ $font-sizes: (
     height: 56px;
   }
 }
+
+.topic-nav-hero {
+  z-index: -1;
+  width: 100%;
+  height: 490px;
+  object-fit: cover;
+  object-position: 80% 80%;
+}
+
+.topic-nav-hero-title {
```

Comment:
This is a tricky one since we don't have any other finalized sections yet, but I was imaging that this would be covered under something like:

```
h3 (or something equiv) {
   font-family: 'Aleo', ...
   font-size: 40px;
}
```
because we should be consistent in our use of h1 ~ h5 and their corresponding styles right?
Makes me think that maybe I need to push in my bootstrap vars card even if it's only tangentially related, but more so because I can see us updating and adding core stuff like this as we go along with each page

Reply:
I agree, I think ideally that'd be the case. Maybe we can ask Lydia to specifically keep things consistent across pages (i.e. have set styles that have set sizes at different screen sizes)

Reply:
+1